PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -318,22 +318,31 @@ def miniAOD_customizeCommon(process):
     for idmod in photon_ids:
         setupAllVIDIdsInModule(process,idmod,setupVIDPhotonSelection,None,False,task)
 
-    #---------------------------------------------------------------------------
-    #Adding  Boosted Subjets taus
+    #-- Adding boosted taus
     from RecoTauTag.Configuration.boostedHPSPFTaus_cfi import addBoostedTaus
     addBoostedTaus(process)
-    #---------------------------------------------------------------------------
-    #Adding tau reco for 80X legacy reMiniAOD
-    #make a copy of makePatTauTask to avoid labels and substitution problems
-    _makePatTausTaskWithTauReReco = process.makePatTausTask.copy()
-    #add PFTau reco modules to cloned makePatTauTask
     process.load("RecoTauTag.Configuration.RecoPFTauTag_cff")
-    _makePatTausTaskWithTauReReco.add(process.PFTauTask)
-    #replace original task by extended one for the miniAOD_80XLegacy era
+    process.load("RecoTauTag.Configuration.HPSPFTaus_cff")
+    #-- Adding customization for 94X 2017 legacy reMniAOD
+    from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
+    _makePatTausTaskWithRetrainedMVATauID = process.makePatTausTask.copy()
+    _makePatTausTaskWithRetrainedMVATauID.add(process.hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTTask)
+    process.hpsPFTauDiscriminationByVVLooseIsolationMVArun2v1DBoldDMwLT = process.hpsPFTauDiscriminationByVLooseIsolationMVArun2v1DBoldDMwLT.clone()
+    process.hpsPFTauDiscriminationByVVLooseIsolationMVArun2v1DBoldDMwLT.mapping[0].cut = cms.string("RecoTauTag_tauIdMVAIsoDBoldDMwLT2017v1_WPEff95")
+    _makePatTausTaskWithRetrainedMVATauID.add(process.hpsPFTauDiscriminationByVVLooseIsolationMVArun2v1DBoldDMwLT)
```

Comment:
please add this module to hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTTask in RecoTauTag/Configuration/python/HPSPFTaus_cff.py
This will  minimize required knowledge of the tau reco in the miniAOD_tools.
(this was a part of my https://github.com/cms-sw/cmssw/pull/21977#issuecomment-360923538 
``` define hpsPFTauIsolationMVArun2v1DBoldDMwLTTask in in HPSPFTaus_cff.py with 7 needed modules;
    add hpsPFTauDiscriminationByVVLooseIsolationMVArun2v1DBoldDMwLT only for this era```
although the text was probably not explicit enough to say that the latter action also should be applied in HPSPFTaus_cff.py